website/docs/foundations/elevation/partials/code/how-to-use.md

```diff
@@ -1,47 +1,52 @@
 ## How to use these styles
 
-There are two different ways to apply an "elevation" or "surface" effect to a UI element, via **design tokens** or via **CSS helper classes**.
+You can apply an `elevation` or `surface` effect to an element via **design tokens** or **CSS helper classes**.
 
-Which one to use will depend on the context: since both refer to the same design token values, from the design system perspective there is no difference between the two methods (it’s more of a preference for the "consumer" codebase).
+!!! Info
+
+Note that `border-radius` is not included with this token and needs to be set according to the specs of the UI element.
```

Comment:
In the other update docs (I think Focus ring?) there was a point about recommending `inherit` for the border radius value. Is that important/relevant to also note here?

Reply:
It wasn't included in the initial documentation, so I figured it may not be the same recommendation here. @didoo, is using `inherit` recommended here as well?

Reply:
In this case, unlike the focus ring, there may be no "parent" elements that have a border radius. This means the consumer may need to apply the border radius directly to the same element that has the elevation applied to it. So in some cases they can use `inherit`, in others no. Better to keep it generic.